src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -127,8 +127,7 @@ export function Autosuggest<S>(props: Props<S>) {
              * Prevent the default Downshift handler behavior
              * That need for submitting form
              */
-            // eslint-disable-next-line no-param-reassign, dot-notation
-            event.nativeEvent['preventDownshiftDefault'] = true;
+            (event.nativeEvent as any).preventDownshiftDefault = true;
```

Comment:
This is a temporary workaround: tests would issue an error without.

Reply:
temporally, meaning that you are still working on it?

Reply:
No, these instances I commented on I wasn't sure about how to solve them properly. Would appreciate your input, you're probably more familiar with the code.

Reply:
@eszthoff bump

Reply:
I think your solution here is ok